x-pack/plugins/ml/server/routes/anomaly_detectors.ts

```diff
@@ -175,9 +176,28 @@ export function jobRoutes({ router, mlLicense }: RouteInitialization) {
     mlLicense.fullLicenseAPIGuard(async (context, request, response) => {
       try {
         const { jobId } = request.params;
+        const body: Partial<Job> = request.body;
+
+        // if annotations is enabled but model plot is not
+        if (
```

Comment:
we shouldn't have additional logic in this endpoint. this should be a simple proxy to the es endpoint and should behave in the exact same way.

Reply:
Updated in 1b54be9